www/src/pages/foundations/responsive.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DataTable, Container, breakpoints } from '~paragon-react'; // eslint-disable-line
+import SEO from '../../components/SEO';
+import Layout from '../../components/PageLayout';
+
+const getBreakpointDescription = (breakpoint) => {
+  switch (breakpoint) {
+    case 'extraSmall':
+      return { name: 'Extra small', identifier: 'xs' };
+    case 'small':
+      return { name: 'Small', identifier: 'sm' };
+    case 'medium':
+      return { name: 'Medium', identifier: 'md' };
+    case 'large':
+      return { name: 'Large', identifier: 'lg' };
+    case 'extraLarge':
+      return { name: 'Extra large', identifier: 'xl' };
+    case 'extraExtraLarge':
+      return { name: '2Extra large', identifier: 'xxl' };
+    default:
+      return {};
+  }
+};
+
+const IdentifierCell = ({ row }) => <code>{row.values.identifier}</code>;
+const MinWidthCell = ({ row }) => <code>{row.values.minWidth ? `${row.values.minWidth}px` : '-'}</code>;
+const MaxWidthCell = ({ row }) => <code>{row.values.maxWidth ? `${row.values.maxWidth}px` : '-'}</code>;
+
+const Responsive = () => {
+  const breakpointsData = Object.keys(breakpoints).map(breakpoint => {
+    const { minWidth, maxWidth } = breakpoints[breakpoint];
+    const breakpointData = getBreakpointDescription(breakpoint);
+    breakpointData.minWidth = minWidth;
+    breakpointData.maxWidth = maxWidth;
+    return breakpointData;
+  });
+
+  return (
+    <Layout>
+      <Container size="md" className="py-5">
+        {/* eslint-disable-next-line react/jsx-pascal-case */}
+        <SEO title="Responsive" />
+        <h1>Responsive</h1>
+        <h3>Break points</h3>
+        <p>
+          Define the minimum and maximum dimensions at which your layout will change,
+          adapting to different screen sizes, for use in media queries.
+        </p>
+        <h3>CSS Utilities</h3>
+        <DataTable
+          className="pgn-doc__spacing-table"
+          data={breakpointsData}
+          columns={[
+            { Header: 'Name', accessor: 'name' },
+            { Header: 'Identifier', accessor: 'identifier', Cell: IdentifierCell },
+            { Header: 'Min width', accessor: 'minWidth', Cell: MinWidthCell },
+            { Header: 'Max Width', accessor: 'maxWidth', Cell: MaxWidthCell },
+          ]}
+        >
+          <DataTable.Table />
+        </DataTable>
+      </Container>
+    </Layout>
+  );
```

Comment:
> we don't have possibility to let a user override $grid-breakpoints

@monteri Sounds good on not displaying `$grid-breakpoints`. Also, a clarification:

All the default variables defined in Paragon's variables.scss files can be overridden by applying a new theme. For example, the theme `@edx/brand-edx.org` doesn't currently override the `$grid-breakpoints`, simply using the default value provided by Paragon, but it could override these: https://github.com/edx/brand-edx.org/blob/master/paragon/_variables.scss#L329-L335

Per the [documentation](https://paragon-openedx.netlify.app/guides/installation-and-usage):

> When working with a theme the order of imports is important: if you need to override Paragon's variables you must import theme's variables before Paragon core (that's because all variables in Paragon use !default flag which allows you to override them before importing, you can read more [here](https://sass-lang.com/documentation/variables#default-values))

When the `@edx/brand-edx.org` theme's variables are imported before the core Paragon styles, the variables defined in the theme are used in place of the default Paragon ones.